### ISSUE TITLE
[Doc]: Update README

### ISSUE THREAD

**Issue by Tayebsed93** [Orange-OpenSource/ouds-flutter#186]
### Documentation Location

The README of the library needs an update to reflect recent changes and improve the clarity of the information provided. The following points should be addressed:

1. Installation: Add updated installation instructions.
2. Usage Examples: Include additional examples to illustrate the features.
3. Contributions: Clarify the contribution process for interested developers.
4. Licenses and Legal Notices: Check and update the licensing sections if necessary.

### Expected Correction
Improve the documentation to facilitate the use and contribution to the library.

**Comment by B3nz01d** [comment]
Completed during the sprint 18
